lib/Target/OpenFhePke/OpenFhePkeEmitter.cpp

```diff
@@ -171,6 +196,23 @@ LogicalResult OpenFhePkeEmitter::printOperation(func::FuncOp funcOp) {
   return success();
 }
 
+LogicalResult OpenFhePkeEmitter::printOperation(func::CallOp op) {
+  if (op.getNumResults() > 1) {
+    return emitError(op.getLoc(), "Only one return value supported");
+  }
+
+  if (op.getNumResults() != 0) {
+    os << variableNames->getNameForValue(op.getResult(0)) << " = ";
```

Comment:
@ZenithalHourlyRate - I had cherry-picked this PR and was using it to work on a demo, and then realized that this doesn't print out the type. I had to replace with `emitAutoAssignPrefix(result);`